cmd/brew-services.rb

```diff
@@ -236,6 +237,33 @@ def restart(target)
       end
     end
 
+    # "load" a plist
+    def launchctl_load(plist, _function, service)
+      if root?
+        domain_target = "system"
+      else
+        domain_target = "gui/#{Process.uid}"
+      end
+
+      if MacOS.version < :yosemite
```

Comment:
Pedantry but make this the `else` so the first block people see is the current version.